2048/2048/src/main/java/com/appcoins/eskills2048/PlayerRankingAdapter.java

```diff
@@ -0,0 +1,69 @@
+package com.appcoins.eskills2048;
+
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.appcoins.eskills2048.model.User;
+
+import java.util.List;
+
+public class PlayerRankingAdapter extends RecyclerView.Adapter<PlayerRankingAdapter.ViewHolder> {
+  private List<User> localDataSet;
+  private final Context context;
+
+  public static class ViewHolder extends RecyclerView.ViewHolder {
+    private final TextView userNameTextView;
+    private final TextView userScoreTextView;
+
+    public ViewHolder(View view) {
+      super(view);
+      userNameTextView = (TextView) view.findViewById(R.id.user_name);
+      userScoreTextView = (TextView) view.findViewById(R.id.user_score);
+    }
+
+    public TextView getUserNameTextView() {
+      return userNameTextView;
+    }
+
+    public TextView getUserScoreTextView() {
+      return userScoreTextView;
+    }
+  }
+
+  public PlayerRankingAdapter(Context context, List<User> dataSet) {
+    localDataSet = dataSet;
+    this.context = context;
+  }
+
+  @Override
+  public ViewHolder onCreateViewHolder(ViewGroup viewGroup, int viewType) {
+    View view = LayoutInflater.from(viewGroup.getContext())
+        .inflate(R.layout.player_row_item, viewGroup, false);
+
+    return new ViewHolder(view);
+  }
+
+  @Override
+  public void onBindViewHolder(ViewHolder viewHolder, final int position) {
+    viewHolder.getUserNameTextView().setText(localDataSet.get(position).getUserName());
```

Comment:
```suggestion
    viewHolder.bind(localDataSet.get(position));
```

Reply:
bind() is not available on this viewHolder. what should i do?